app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainFragment.kt

```diff
@@ -28,7 +26,7 @@ class MainFragment : Fragment() {
     }
 
     lateinit var mFusedLocationProviderClient: FusedLocationProviderClient
-    val locationRequestId = 100
+    val LOCATION_REQUEST_ID = 100
```

Comment:
Try out the SCREAMING_SNAKE_CASE for deeply immutable data! (https://kotlinlang.org/docs/coding-conventions.html#property-names)

Reply:
just look at https://kotlinlang.org/docs/coding-conventions.html#property-names , good info.

Reply:
That's often used for constants, like MONTHS_IN_YEAR = 12.
val: well, they are immutable, but are they necessarily constants?  I suppose that depends on the use.
Since this does not have the 'const' modifier, I'd likely leave it as lowerCamelCase... and the documentation you provided (excellent source) indicates that as well.

Reply:
OK will Leave as is for now,  but may revisit this later most likely.